lib/viewing/Layout.js

```diff
@@ -82,10 +82,6 @@ var Layout = View.extend({
         return html;
     },
 
-    hideContent: function() {
```

Comment:
I can't find anything that calls it. I think it was part of a failed attempt to optimize transitions between pages on sites with flash ads.my guess is we just forgot to clean it up. Maybe someone else remembers a use case?